react/components/MiniCartContent.js

```diff
@@ -153,29 +181,35 @@ class MiniCartContent extends Component {
     )
 
     const discount = this.calculateDiscount(items, orderForm.value)
-    const { onClickProduct } = this.props
 
     return (
       <Fragment>
         <div className={classes}>
           {items.map(item => (
-            <MiniCartItem
-              {...item}
-              key={item.id}
-              large
-              removeItem={this.onRemoveItem}
-              updateItem={this.onUpdateItems}
-              showRemoveButton={showRemoveButton}
-              showSku={showSku}
-              enableQuantitySelector={enableQuantitySelector}
-              maxQuantity={maxQuantity}
-              onClickProduct={onClickProduct}
-            />
+            <Fragment key={item.id}>
+              <div className="relative">
+                <div className="fr absolute bottom-0 right-0">
+                  <Button icon variation="tertiary" onClick={e => this.onRemoveItem(item.id, e)}>
+                    <IconDelete size={15} color="silver" />
```

Comment:
`silver` ou `c-muted...` of some sort?

Reply:
The correct is c-muted-3, and i try to use it. however, this class is not recognized within the minicart